cmd/process-agent/collector_api_test.go

```diff
@@ -351,8 +367,9 @@ func TestSendPodMessageSendManifestPayload(t *testing.T) {
 func TestSendPodMessageNotSendManifestPayload(t *testing.T) {
 	clusterID, cfg, check := getPodCheckMessage(t)
 
-	init := func(c *Collector) {
+	init := func(c *Collector, s *submitter) {
```

Comment:
Similar comment as above - maybe we could use mock config here?
```
config.Set("orchestrator_explorer.enabled", true)
```

Reply:
Makes sense! Updated